### ISSUE TITLE
Tooltip that label the trigger 

### ISSUE THREAD

**Issue by srapilly** [radix-ui/primitives#1952]
## Feature request

Tooltip that label the control trigger element

### Overview

The current tooltip will always use `aria-describedby`, so it works like a **descriptive** tooltip. In some case we want the control to be **labelled** by the tooltip.

An example can be seen directly in the Tooltip example https://www.radix-ui.com/docs/primitives/components/tooltip

<img width="252" alt="image" src="https://user-images.githubusercontent.com/85184231/218559892-17fe3f33-5be7-4f62-b078-a658fb8cb20c.png">

The button should be labelled by the tooltip. Currently it will always use an `aria-describedby`. 

### Examples in other libraries

https://mui.com/material-ui/api/tooltip/ 

In their component, by default the tooltip will label the controls, they have a props `describeChild` that will changes the `aria-labelledby` to an `aria-describedby`

### Who does this impact? Who is this for?

When using a component that should be labelled by the tooltip (for example an icon button) 

### Proposal

maybe a new boolean props  `labelChild` ? 

I could help with the implementation of this feature :)

**Comment by benoitgrelard** [comment]
Hey @srapilly,

I believe we already take care of this, albeit a bit differently.
If we `aria-labelledby` instead of `aria-describedby`, it would skip the button text entirely so that's not really good.
In cases, where you want to label a trigger that doesn't have text (like an icon button as you say) if you set an `aria-label` on the trigger, we make sure it takes priority for screen readers.

See here:

https://github.com/radix-ui/primitives/blob/6ad72cb90e7820182de778028d4acd1f006999c6/packages/react/tooltip/src/Tooltip.tsx#L556

We have an example in our storybook too:

https://github.com/radix-ui/primitives/blob/6ad72cb90e7820182de778028d4acd1f006999c6/packages/react/tooltip/src/Tooltip.stories.tsx#L510-L528

I will close the issue as I think we are covering the basis nicely already.

Hope that makes sense.

✌️
